run.cmd

```diff
@@ -21,7 +21,7 @@ if defined VisualStudioVersion goto :Run
 
 set _VSWHERE="%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe"
 if exist %_VSWHERE% (
-  for /f "usebackq tokens=*" %%i in (`%_VSWHERE% -latest -property installationPath`) do set _VSCOMNTOOLS=%%i\Common7\Tools
+  for /f "usebackq tokens=*" %%i in (`%_VSWHERE% -latest -prerelease -property installationPath`) do set _VSCOMNTOOLS=%%i\Common7\Tools
```

Comment:
Would you mind putting it in the other place we call vswhere as well? https://github.com/dotnet/corefx/blob/dc49365a1b9bd707b4e5b5b29cef04943e811b98/src/Native/build-native.cmd#L61

Reply:
updated